packages/components/organisms/f-cookie-banner/src/components/ReopenBannerLink.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <a
+        href="#"
+        :class="['o-link--bold', 'o-link--full', $style['reopen-link'], { [$style['reopen-link--greyBg']]: useGreyBackground }]"
+        :use-grey-background="useGreyBackground"
+        @click.prevent="reopenBanner">{{ message }}
+    </a>
+</template>
+
+<script>
+export default {
+    name: 'ReopenBannerLink',
+    props: {
+        message: {
+            type: String,
+            default: ''
+        },
+        useGreyBackground: {
+            type: Boolean,
+            required: true
+        }
+    },
+    methods: {
+        reopenBanner () {
+            this.$emit('reopenBanner');
+        }
+    }
+};
+</script>
+
+<style module lang="scss">
+a.reopen-link {
```

Comment:
```suggestion
.reopen-link {
```
if it's safe to drop the tag here I would do that. It will be slightly faster and more flexible.